README.md

```diff
@@ -1,3 +1,24 @@
-# Altanic
+# ![Altanic](sprites/github/altanic.png)
 
 My never finished "project" that has been archived and never finished since always.
+I was trying to make this as good readable as I can.
+
+***Requires multicrafter-lib and I do not thing that this can be qualified as js mod***
+
+## Dla wszystkich polaczków jak ja
+
+Tłumaczenie contentu wychodzi razem z jego utworzeniem, a bynajmniej taki mam zamiar.
+Nie wiem konkrentie, ale chyba zrobię również polski opis tego co w kodzie, ale zakładm, iż ignlisz szprechacać umicie.
+Jak coś to `#polski` na discordzie mindustry albo pisać, byle nie dzownić, do `[PL][KT][ZPGM]Swirek3331#0611`.
+
+## Content
+
+I have came to conclusion, that beter will be if a new content will be developt in other branch, but some importatn bug fixes.
+
+## **Other**
+
+My discord username and tag: `[PL][KT][ZPGM]Swirek3331#0611`
+
+[Multicrafter-lib](https://liplum.github.io/MultiCrafterLib/)
+
+[Multicrafter-lib-repository](https://github.com/liplum/MultiCrafterLib)
```

Comment:
When I open a preview in vsc, the heading is being changed, but I do not see the same result on github.
And yes, fuck regulation.